integreat_compass/cms/forms/offers/report_form.py

```diff
@@ -0,0 +1,18 @@
+from django import forms
+
+from ...models import Report
+
+
+class ReportForm(forms.ModelForm):
+    """
+    Form for reporting an offer
+    """
```

Comment:
```suggestion
from django import forms

from ...models import Report
from ..custom_model_form import CustomModelForm


class ReportForm(CustomModelForm):
    """
    Form for reporting an offer
    """
```
No reason not to use then fancy custom class we already use everywhere else 😄